packages/utils/src/index.ts

```diff
@@ -121,5 +120,4 @@ export {
   utcToLocal,
   validationDataMerge,
   withIdRefPrefix,
-  base64,
```

Comment:
Can you also remove this from the [documentation](https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/utility-functions/#base64) for the `utils` functions?